src/adm/tiles.rs

```diff
@@ -232,10 +233,13 @@ pub async fn get_tiles(
                 .await
                 .map_err(|e| {
                     // ADM servers are not returning correct information
-                    HandlerErrorKind::BadAdmResponse(format!(
+                    let mut e: HandlerError = HandlerErrorKind::BadAdmResponse(format!(
                         "ADM provided invalid response: {:?}",
                         e
                     ))
+                    .into();
+                    e.tags = tags.clone();
```

Comment:
These tags on `BadAdmResponse` will be ignored because the handler uses a different set of tags when reporting `BadAdmResponse`.

I think we should kill that handler behavior entirely and just propagate the errors to the client. It happens [rarely](https://earthangel-b40313e5.influxcloud.net/explore?orgId=1&left=%5B%22now-90d%22,%22now%22,%22svcops%22,%7B%22refId%22:%22A%22,%22policy%22:%22default%22,%22resultFormat%22:%22time_series%22,%22orderByTime%22:%22ASC%22,%22tags%22:%5B%7B%22key%22:%22realm%22,%22value%22:%22prod%22,%22operator%22:%22%3D%22%7D%5D,%22groupBy%22:%5B%7B%22type%22:%22time%22,%22params%22:%5B%22$__interval%22%5D%7D,%7B%22type%22:%22fill%22,%22params%22:%5B%22null%22%5D%7D%5D,%22select%22:%5B%5B%7B%22type%22:%22field%22,%22params%22:%5B%22value%22%5D%7D,%7B%22type%22:%22sum%22,%22params%22:%5B%5D%7D%5D%5D,%22measurement%22:%22contile.tiles.invalid%22%7D%5D) anyway and then this would work as expected.

Reply:
Ah, crap. You're right. Forgot about the special handler.

Reply:
Not sure we can get rid of that clause, since it's storing an empty record to prevent us from stampeding the ADM servers if there's an error. I think we still want that. I'll comment and clean it up a bit.

Reply:
If clients stampede contile due to it returning 5xx I think we're in trouble (AFAIK they don't do this -- we concluded at some point that the clients handle errors ok so there's really no need to mask them).

I might end up changing this around w/ the fallback protocol handling anyway though, so this is fine as is.